openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -1448,3 +1376,254 @@ def update_plot(ii):
     plt.show()
     return line_ani
 
+
+
+
+
+# =============================================================================
+# Experimental Functions
+# =============================================================================
+
+def _plot_on_click(self, event):
```

Comment:
Is this needed? I see this function is present in ``class plot_deformedshape_events``

Reply:
It was not no.